Tests/Responses/TransferMethodConfigurationFieldsResponse.json

```diff
@@ -27,7 +27,7 @@
                         {
                             "category": "ACCOUNT",
                             "dataType": "NUMBER",
-                            "isRequired": true,
+                            "isRequired": false,
```

Comment:
why did you change it to false? This field is always true

Reply:
just for test

Reply:
it is fake data and we can check not only for true

Reply:
i will revert